src/js/media-container.js

```diff
@@ -276,19 +277,23 @@ class MediaContainer extends window.HTMLElement {
       );
     }
 
+    // Assume user is inactive until they're not (aka user-inactive by default is true)
+    // This allows things like autoplay and programmatic playing to also initiate hiding controls (CJP)
+    this.setAttribute('user-inactive', 'user-inactive');
+
     const scheduleInactive = () => {
       this.removeAttribute('user-inactive');
       const evt = new window.CustomEvent(
         MediaStateChangeEvents.USER_INACTIVE, 
         { composed: true, bubbles: true, detail: false }
       );
       this.dispatchEvent(evt);
-      window.clearTimeout(this.inactiveTimeout);
```

Comment:
renamed this per Media Chrome naming conventions to indicate this should be treated as a private variable.